DependencyInjection/Compiler/MessengerPass.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+/**
+ * Class for Symfony Messenger component integrations
+ */
+class MessengerPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        // Remove wired services if the Messenger component actually isn't enabled:
+        if (!$container->hasAlias('message_bus') || !is_subclass_of($container->findDefinition('message_bus')->getClass(), MessageBusInterface::class, true)) {
```

Comment:
The 3rd argument of `is_subclass_of()` is already `true` by default, right?

Reply:
`true` :)

PHP API consistency (see [`is_a`](http://php.net/manual/fr/function.is-a.php)) tricked me!